src/ImageProcessor/Imaging/MetaData/ExifPropertyTagConstants.cs

```diff
@@ -0,0 +1,87 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ExifPropertyTagConstants.cs" company="James Jackson-South">
+//   Copyright (c) James Jackson-South.
+//   Licensed under the Apache License, Version 2.0.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+
+namespace ImageProcessor.Imaging.MetaData
+{
+    /// <summary>
+    /// Contains constants grouping together common property items.
+    /// </summary>
+    public class ExifPropertyTagConstants
+    {
+        /// <summary>
+        /// Gets all required property items. The Gif format specifically requires these properties.
+        /// </summary>
+        public static readonly ExifPropertyTag[] RequiredPropertyItems = {
+            ExifPropertyTag.LoopCount, ExifPropertyTag.FrameDelay
+        };
+
+        /// <summary>
+        /// Gets all required property items plus geolocation specific property items. 
+        /// </summary>
+        public static readonly ExifPropertyTag[] GeolocationPropertyItems = RequiredPropertyItems.Union(new[]{
+            ExifPropertyTag.GpsAltitude,
+            ExifPropertyTag.GpsAltitudeRef,
+            ExifPropertyTag.GpsDestBear,
+            ExifPropertyTag.GpsDestBearRef,
+            ExifPropertyTag.GpsDestDist,
+            ExifPropertyTag.GpsDestDistRef,
+            ExifPropertyTag.GpsDestLat,
+            ExifPropertyTag.GpsDestLatRef,
+            ExifPropertyTag.GpsDestLong,
+            ExifPropertyTag.GpsDestLongRef,
+            ExifPropertyTag.GpsGpsDop,
+            ExifPropertyTag.GpsGpsMeasureMode,
+            ExifPropertyTag.GpsGpsSatellites,
+            ExifPropertyTag.GpsGpsStatus,
+            ExifPropertyTag.GpsGpsTime,
+            ExifPropertyTag.GpsIFD,
+            ExifPropertyTag.GpsImgDir,
+            ExifPropertyTag.GpsImgDirRef,
+            ExifPropertyTag.GpsLatitude,
+            ExifPropertyTag.GpsLatitudeRef,
+            ExifPropertyTag.GpsLongitude,
+            ExifPropertyTag.GpsLongitudeRef,
+            ExifPropertyTag.GpsMapDatum,
+            ExifPropertyTag.GpsSpeed,
+            ExifPropertyTag.GpsSpeedRef,
+            ExifPropertyTag.GpsTrack,
+            ExifPropertyTag.GpsTrackRef,
+            ExifPropertyTag.GpsVer
+        }).ToArray();
+
+        /// <summary>
+        /// Gets all required property items plus copyright and geolocation specific property items. 
```

Comment:
"and gelocation" needs removing from the comment here.

Other than that looks pretty solid to me. 👍